docs/content/action/config.md

```diff
@@ -3,72 +3,67 @@ title: Config
 sort: 1
 ---
 
-# Metadata "config"
+# Configuration properties
 
-Config are used to contextualise the handler function.
+Configuration properties are used to contextualize the handler function.
 
 ## Usage
 
-They are usually provided as an object when calling an action. For example, the `system.execute` action can receive on object with a "cmd" property:
+They are usually provided as an object when calling an action. For example, the `nikita.execute` action can receive an object with the `command` property:
 
 ```js
 nikita
-// Object with "cmd" config
-.system.execute({cmd: 'whoami'})
+// Object with "command" config
+.execute({command: 'whoami'})
 ```
 
 ## Short declaration
 
-The `system.execute` action can also receive the command as a string. This declaration achieve the same result as the previous example:
+The `nikita.execute` action can also receive the command as a string. This declaration achieves the same result as the previous example:
 
 ```js
 nikita
 // Command as a string
-.system.execute('whoami');
+.execute('whoami')
 ```
 
-The string config in the previous example is here for conveniency. Internally, the execute handler receives config as an object and search for the "argument" config. Here's an example:
+The string configuration in the previous example is here for convenience. Internally, the execute handler receives the configuration as an object and search for the `argument` metadata property. Here's an example:
 
 ```js
 nikita
-.register('execute', function({config}, callback){
-  config.cmd = config.argument if typeof config.argument is 'string'
+// Register the action
+.registry.register('execute', ({config, metadata}) => {
+  // Check the argument
+  if (metadata.argument != null)
+    config.command = metadata.argument
   // More code goes here
-});
-.execute('whoami', function(err, {stdout}){
-  console.info('I am ' + stdout.trim());
 })
```

Comment:
I think using `execute` and reimplementing it is confusing. Do you agree or shall we leave it like this ?

Reply:
agree